Tests/FluentAssertions.Specs/Collections/CollectionAssertionSpecs.BeEquivalentTo.cs

```diff
@@ -158,6 +158,50 @@ public void Default_immutable_lists_should_be_equivalent()
             // Act / Assert
             collection.Should().BeEquivalentTo(collection1);
         }
+
+        [Fact]
+        public void Succeed_for_collection_with_case_different_strings_when_IgnoringCase()
+        {
+            // Arrange
+            var actual = new[] { "first", "test", "last" };
+            var expect = new[] { "first", "TEST", "last" };
```

Comment:
🙃 We prefer to name this the `expectation`

Reply:
Done!